UnicornRender/video/source/CameraFpsController.cpp

```diff
@@ -5,71 +5,73 @@
 */
 
 #include <unicorn/video/CameraFpsController.hpp>
+
 #include <algorithm>
 
 namespace unicorn
 {
 namespace video
 {
-CameraFpsController::CameraFpsController(unicorn::video::Camera& camera)
-    : sensitivity(0.1f), 
-    speed(100.0f), 
-    m_lastX(0.0), 
-    m_lastY(0.0), 
+
+CameraFpsController::CameraFpsController(glm::mat4& cameraView)
+    : CameraController(cameraView),
+    sensitivity(0.1f),
+    m_lastX(0.f),
+    m_lastY(0.f),
     m_yaw(90.0),
-    m_pitch(0.0), 
-    m_dirty(false),
-    m_camera(camera)
+    m_pitch(0.0),
+    m_dirty(true)
```

Comment:
put all commas before each member so that when you add more variables, diff will be shorter

```cpp
: CameraController(cameraView)
, sensitivity(0.1f)
, m_lastX(0.f)
, m_lastY(0.f)
, m_yaw(90.0)
, m_pitch(0.0)
, m_dirty(true)
```